target/snitch_cluster/sw/tests/snax-run.yaml

```
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+
+runs:
+  - app: atomics
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x4
+  - app: barrier
+  - app: data_mover
+  - app: dma_simple
+  #- app: event_unit # SNAX failes because this test needs more cores
+  - app: fence_i
+  - app: fp8_comparison_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with segfault
+  - app: fp8_comparison_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with segfault
+  - app: fp8_computation_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with JIT issue
+  - app: fp8_computation_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x6
+  - app: fp8alt_comparison_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with segfault
+  - app: fp8alt_comparison_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x10
+  - app: fp8alt_computation_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with JIT issue
+  - app: fp8alt_computation_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x12
+  - app: fp16_comparison_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x10
+  - app: fp16_comparison_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x10
+  - app: fp16_computation_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with JIT issue
+  - app: fp16_computation_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x6
+  - app: fp16alt_comparison_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x10
+  - app: fp16alt_comparison_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x10
+  - app: fp16alt_computation_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with JIT issue
+  - app: fp16alt_computation_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x16
+  - app: fp32_comparison_scalar
+  - app: fp32_comparison_vector
+  - app: fp32_computation_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x2
+  - app: fp32_computation_vector
+    simulators: [vsim, vcs, verilator] # banshee fails with exit code 0x2
+  - app: fp32_conversions_scalar
+    simulators: [vsim, vcs, verilator] # banshee fails with illegal instruction
+  - app: fp64_conversions_scalar
+    simulators: [vsim, vcs, verilator]
+  # - app: interrupt
+  - app: interrupt-local
+  - app: multi_cluster
+  #- app: openmp_parallel # This requires multiple cores too
+  #- app: openmp_for_static_schedule # This requires multiple cores too
+  - app: openmp_double_buffering
+  - app: perf_cnt
+  - app: printf_simple
+  - app: printf_fmtint
+  - app: simple
+  #- app: team_global # This requires all 9 cores to be present
+  - app: tls
+  - app: varargs_1
+  - app: varargs_2
+  - app: zero_mem
+  - app: non_null_exitcode
```

```suggestion
  - app: non_null_exitcode
  - app: snax-mac
    simulators: [vsim, vcs, verilator]
```
Don't forget to include your own test!
I don't think this is supposed to work in banshee right?

---

Let's test it in another test 🛩️  hehe but most likely not since the SNAX was not modelled anyway. I'm not sure if their banshee setup actually builds for a new architecture too.